app/src/layouts/tabular/tabular.vue

```diff
@@ -7,7 +7,7 @@
 			v-model:headers="tableHeadersWritable"
 			class="table"
 			fixed-header
-			:show-select="readonly ? false : selection !== undefined"
+			:show-select="showSelect ? showSelect : selection !== undefined"
```

Comment:
Should we still include `readonly` in the logic here? Honestly not super sure when does "readonly" actually takes effect 🤔

Reply:
Hey @azrikahar 
Thank you for your review 🤝

It seems this `readonly` is only being used here:
https://github.com/directus/directus/blob/5d23dfd37f8b75039c0759820b0373a7ba459f97/app/src/modules/settings/routes/presets/item.vue#L10

I believe `readonly` could be used to prevent clicks on rows while `show-select` changes the visibility of checkbox.
IMO, it should be possible to have checkboxes but not be able to click on rows. The item in Presets is an example of this scenario.

Reply:
ohh that makes perfect sense. Thanks for clarifying!